src/views/table_element_footer.ts

```diff
@@ -34,6 +37,28 @@ export class TableElementFooterView<PropsType extends BaseNodeProps>
     this.contentDOM = document.createElement('div')
     this.dom.appendChild(this.contentDOM)
   }
-}
 
+  // This alternative solution does work, but sometimes I noticed that while it successfully sets the class name directly after any change (approve, reject, etc.), the style isn't reflected until I click on the editing area. This behavior seems unusual to me, although it doesn't occur consistently.
```

Comment:
In any case it sounds like there is a minor bug with the update not being applied properly, so nothing what would make that approach inviable.